eva-accession-import-dbsnp2/src/test/java/uk/ac/ebi/eva/accession/dbsnp2/test/BatchTestConfiguration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp2.test;
+
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.batch.BatchProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import uk.ac.ebi.eva.accession.core.configuration.MongoConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ChunkSizeCompletionPolicyConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ImportDbsnpJsonVariantsJobConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ImportDbsnpJsonVariantsProcessorConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ImportDbsnpJsonVariantsReaderConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ImportDbsnpJsonVariantsStepConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.ImportDbsnpJsonFlowConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.configuration.InputParametersConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp2.runner.DbsnpJsonImportVariantsJobLauncherCommandLineRunner;
+import uk.ac.ebi.eva.commons.batch.job.JobExecutionApplicationListener;
+
+@EnableAutoConfiguration
+@Import({MongoConfiguration.class,
+        ImportDbsnpJsonVariantsJobConfiguration.class,
+        ImportDbsnpJsonVariantsStepConfiguration.class,
```

Comment:
The configuration classes should be able to deal with the nesting via `@Import` themselves.